test/suites/standard2/validate-certificate-rotation.robot

```diff
@@ -76,3 +84,40 @@ Certs Should Expire On
     ${expiration_date}=    Command Should Work
     ...    ${OSSL_CMD} ${cert_file} | grep notAfter | cut -f2 -d'=' | awk '{printf ("%s %02d %d",$1,$2,$4)}'
     Should Be Equal As Strings    ${cert_expected_date}    ${expiration_date}
+
+Wait For Multiple Certificates Regeneration
+    [Documentation]    Wait for multiple certificate files to be regenerated and valid
+
+    @{cert_files}=    Create List
+    ...    /var/lib/microshift/certs/kube-control-plane-signer/kube-scheduler/client.crt
+    ...    /var/lib/microshift/certs/kube-control-plane-signer/kube-controller-manager/client.crt
+    ...    /var/lib/microshift/certs/admin-kubeconfig-signer/admin-kubeconfig-client/client.crt
+    FOR    ${cert_file}    IN    @{cert_files}
+        Wait Until Keyword Succeeds    30x    5s
+        ...    Certificate Should Be Valid For Current Time    ${cert_file}
+    END
+
+Certificate Should Be Valid For Current Time
+    [Documentation]    Verify that certificate is valid for the current system time
+    [Arguments]    ${cert_file}
+
+    ${cert_not_before}=    Command Should Work
+    ...    ${OSSL_CMD} ${cert_file} | grep notBefore | cut -f2 -d'=' | xargs -I {} date -d "{}" +%s
+    ${current_time}=    Command Should Work    date +%s
+    Should Be True    ${cert_not_before} <= ${current_time}
+    ...    msg=Certificate NotBefore (${cert_not_before}) is after current time (${current_time})
+
+Kubeconfig Should Be Updated
```

Comment:
the test name doesn't match the documentation,its too generic.

Reply:
thanks, I rewrite the functions, I think it's easier now
can you please review it again?